client/templates/map.js

```diff
@@ -0,0 +1,545 @@
+
+//uses the geojson file from:  https://github.com/singingwolfboy/MBTA-GeoJSON
+
+Template.map.rendered = function() {
+
+  this.subscribe('reports', function() {
+    //this function makes sure DOM updates have occurred
+    Tracker.afterFlush(function() {
+      fetchStopsAndUpdateMap();
+    });
+  });
+
+  var colors = {
+    red: "#C60813",
+    orange: "#FA6820",
+    green: "#0A6536",
+    blue: "#041299",
+    purple: "#801174",
+    silver: "#949494"
+  };
+
+  function chooseColor(d) {
```

Comment:
can you add a comment here about what this function expects as argument and what it returns?